package.json

```diff
@@ -154,10 +154,10 @@
     "whatwg-fetch": "^2.0.3"
   },
   "peerDependencies": {
-    "react": "^16.2.0"
+    "react": "^15.0.0 || ^16.0.0"
   },
   "engines": {
-    "yarn": "^1.3.2",
+    "yarn": "^0.24.0 || ^1.0.0",
```

Comment:
@jmcolella We spoke about this offline. I think we should have locked-in engine versioning.